route/table.go

```diff
@@ -14,6 +14,7 @@ import (
 
 	"github.com/fabiolb/fabio/metrics"
 	"github.com/ryanuber/go-glob"
+	glob2 "github.com/gobwas/glob"
```

Comment:
Please name this `glob` and also remove the `github.com/ryanuber/go-glob` library in a separate commit.